pandas/tests/io/parser/test_parse_dates.py

```diff
@@ -55,6 +55,40 @@ def test_separator_date_conflict(all_parsers):
     )
     tm.assert_frame_equal(df, expected)
 
+@pytest.fixtures(all_parsers)
```

Comment:
you don't need this line at all

Reply:
how to run pre-commit for a single file because running pre-commit run --all-files like in pre-commit website is too slow for my computer